CONTRIBUTING.md

```diff
@@ -35,13 +35,14 @@ Avoid exception handling. Because Hyrise is not a product, we do not have to rec
 
 ## Naming Conventions
 - Files: lowercase separated by underscores, e.g., abstract_operator.cpp
-- Types (classes, structs, enums, typedefs, using): CamelCase starting with uppercase letter, e.g., `BaseColumn`
+- Types (classes, structs, enums, typedefs, using): CamelCase starting with uppercase letter, e.g., `TableScan`
 - Variables: lowercase separated by underscores, e.g., `chunk_size`
-- Functions: lowercase separated by underscores, e.g., `get_num_in_tables()`
-- Private / protected members / methods: like variables / functions with leading underscore, e.g., `_get_chunks()`
-- Classes that are used only to have a non-templated base class are named `BaseXY` (e.g., BaseColumn), while classes that have multiple differing implementations are named `AbstractXY` (e.g., AbstractOperator)
+- Functions: lowercase separated by underscores, e.g., `append_mutable_chunk()`
+- Private / protected members / methods: like variables / functions with leading underscore, e.g., `_on_execute()`
+- Classes that are used only to have a non-templated base class are named `BaseXY` (e.g., `BaseValueSegment`), while classes that have multiple differing implementations are named `AbstractXY` (e.g., `AbstractOperator`)
 - Choose descriptive names. Avoid `i`, `j`, etc. in loops.
 - In cases where a constructor parameter would have the same name as the member it initializes, prefix it with `init`: `C(int init_foo) : foo(init_foo) {}`
+- If an identifier contains a verb or an adjective in addition to a noun, the schema [verb|adjective]\[verb] is preferred, e.g., use `left_input` rather than ~~`input_left`~~ and `set_left_input()` rather than ~~`set_input_left()`~~.
```

Comment:
Why is `[noun]` not part of the schema?

Reply:
Good point, it's a type. It should be `[verb|adjective][noun]`